dat/events/dvaered/warlords_battle.lua

```diff
@@ -133,44 +133,56 @@ function attack ()
    attackers[2*n+7]:memory().shield_return = 99
 
    attackers = arrangeList(attackers)  --The heaviest ships will surround the leader
+   form = formation.random_key()
 
    for i, j in ipairs(attackers) do
       j:rename("Invader")
-      j:setFaction("Thugs")  --I use Thugs and Associates because they won't interact with anybody
+      j:setFaction("Thugs") -- I use Thugs and Associates because they won't interact with anybody
```

Comment:
Might be worth using the new dynamic faction stuff ( https://naev.org/api/modules/faction.html#dynAdd ). I use some in the minerva missions if you want examples.